packtools/sps/validation/article_and_subarticles.py

```diff
@@ -162,7 +162,7 @@ def validate_specific_use(self):
             ]
         """
         try:
-            specific_use_list = self.params["specific_use_list"]
+            specific_use_list = [item for item in self.params["specific_use_list"].keys()]
```

Comment:
@Rossi-Luciano no lugar de `specific_use_list = [item for item in self.params["specific_use_list"].keys()]`, use `list(self.params["specific_use_list"].keys())`

Reply:
OK, @robertatakenaka